### PR TITLE
feat(suite): copy contract address warning

### LINKED ISSUE #12842: EVM Tokens / Fix copy icon alignment + copy warning
Screenshot 2024-06-11 at 17 48 59
Screenshot 2024-06-11 at 18 40 03

On click do not copy directly but show warning with `This is not receive address` `Copy` red button and close button

### PR DESCRIPTION
**Pull request by AdamSchinzel**
## Description

Warn users before copying contract address and educate them that It's not their receive address. Copy is from Chris.

## Related Issue

Resolve #12842

## Screenshots:
Screenshot 2024-06-18 at 9 39 32 AM

### CODE REVIEW

File: packages/suite/src/components/suite/modals/ReduxModal/CopyContractAddress.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            dispatch(notificationsActions.addToast({ type: 'copy-to-clipboard' }));
+        }
+        onCancel();
+    };
+
+    return (
+        <StyledModal
+            isCancelable
+            onCancel={onCancel}
+            icon="warningTriangleLight"
```
Comment (tomasklim) [inline packages/suite/src/components/suite/modals/ReduxModal/CopyContractAddress.tsx:34]: there is cursor `pointer` on icon and it is weird @jvaclavik

**Review by tomasklim — COMMENTED** [review]
LGTM from my side would be nice to polish modal with @jvaclavik 

I think that we should also improve wording in the modal but this is different story and we can do it in crowdin

File: packages/suite/src/components/suite/modals/ReduxModal/CopyContractAddress.tsx (hunk 1 above)
Comment (jvaclavik) [reply packages/suite/src/components/suite/modals/ReduxModal/CopyContractAddress.tsx:34]: @AdamSchinzel I pushed to your PR fix for the cursor bug: https://github.com/trezor/trezor-suite/pull/12934/commits/86194c9d6c551a190e90308687730278c81b59a1

Comment (AdamSchinzel) [reply packages/suite/src/components/suite/modals/ReduxModal/CopyContractAddress.tsx:34]: Thanks

**Review by jvaclavik — APPROVED** [review]
Approving 👍 

just FYI: color of the icon should be yellow and we are going to refactor it in the future, because it needs deeper refactoring.

**Review by tomasklim — APPROVED** [review]

**Pull request merged**